{"message":"Unable to ensure uniqueness for usernames: ","stack":"MongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]\n    at Pool.<anonymous> (/Users/adammarsh/nodeproj/newproject/node_modules/mongodb-core/lib/topologies/server.js:564:11)\n    at Pool.emit (events.js:189:13)\n    at Pool.EventEmitter.emit (domain.js:441:20)\n    at Connection.<anonymous> (/Users/adammarsh/nodeproj/newproject/node_modules/mongodb-core/lib/connection/pool.js:317:12)\n    at Object.onceWrapper (events.js:277:13)\n    at Connection.emit (events.js:189:13)\n    at Connection.EventEmitter.emit (domain.js:441:20)\n    at Socket.<anonymous> (/Users/adammarsh/nodeproj/newproject/node_modules/mongodb-core/lib/connection/connection.js:246:50)\n    at Object.onceWrapper (events.js:277:13)\n    at Socket.emit (events.js:189:13)","name":"MongoNetworkError","errorLabels":["TransientTransactionError"],"level":"warn","timestamp":"2019-03-03T18:58:20.147Z"}
{"message":"Unable to ensure uniqueness for user email addresses: ","stack":"MongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]\n    at Pool.<anonymous> (/Users/adammarsh/nodeproj/newproject/node_modules/mongodb-core/lib/topologies/server.js:564:11)\n    at Pool.emit (events.js:189:13)\n    at Pool.EventEmitter.emit (domain.js:441:20)\n    at Connection.<anonymous> (/Users/adammarsh/nodeproj/newproject/node_modules/mongodb-core/lib/connection/pool.js:317:12)\n    at Object.onceWrapper (events.js:277:13)\n    at Connection.emit (events.js:189:13)\n    at Connection.EventEmitter.emit (domain.js:441:20)\n    at Socket.<anonymous> (/Users/adammarsh/nodeproj/newproject/node_modules/mongodb-core/lib/connection/connection.js:246:50)\n    at Object.onceWrapper (events.js:277:13)\n    at Socket.emit (events.js:189:13)","name":"MongoNetworkError","errorLabels":["TransientTransactionError"],"level":"warn","timestamp":"2019-03-03T18:58:20.152Z"}
{"message":"Unable to ensure uniqueness for role name: ","stack":"MongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]\n    at Pool.<anonymous> (/Users/adammarsh/nodeproj/newproject/node_modules/mongodb-core/lib/topologies/server.js:564:11)\n    at Pool.emit (events.js:189:13)\n    at Pool.EventEmitter.emit (domain.js:441:20)\n    at Connection.<anonymous> (/Users/adammarsh/nodeproj/newproject/node_modules/mongodb-core/lib/connection/pool.js:317:12)\n    at Object.onceWrapper (events.js:277:13)\n    at Connection.emit (events.js:189:13)\n    at Connection.EventEmitter.emit (domain.js:441:20)\n    at Socket.<anonymous> (/Users/adammarsh/nodeproj/newproject/node_modules/mongodb-core/lib/connection/connection.js:246:50)\n    at Object.onceWrapper (events.js:277:13)\n    at Socket.emit (events.js:189:13)","name":"MongoNetworkError","errorLabels":["TransientTransactionError"],"level":"warn","timestamp":"2019-03-03T18:58:20.153Z"}
